<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DeviceWidget</class>
 <widget class="QWidget" name="DeviceWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>306</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="iconImage">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="boldNameLabel">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="nameLabel">
       <property name="text">
        <string>Device Title</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QToolButton" name="muteToggleButton">
       <property name="toolTip">
        <string>Mute audio</string>
       </property>
       <property name="icon">
        <iconset theme="audio-volume-muted"/>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="lockToggleButton">
       <property name="toolTip">
        <string>Lock channels together</string>
       </property>
       <property name="icon">
        <iconset theme="changes-prevent-symbolic"/>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="defaultToggleButton">
       <property name="toolTip">
        <string>Set as fallback</string>
       </property>
       <property name="icon">
        <iconset theme="applications-multimedia"/>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QWidget" name="portSelect" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="0,1">
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>&lt;b&gt;Port:&lt;/b&gt;</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="portList"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QGridLayout" name="channelsGrid">
     <property name="horizontalSpacing">
      <number>0</number>
     </property>
    </layout>
   </item>
   <item>
    <widget class="QCheckBox" name="advancedOptions">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>Show advanced options</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="advancedWidget" native="true">
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>9</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QWidget" name="encodingSelect" native="true">
        <layout class="QGridLayout" name="gridLayout">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <property name="spacing">
          <number>0</number>
         </property>
         <item row="0" column="0">
          <widget class="QCheckBox" name="encodingFormatPCM">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>PCM</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QCheckBox" name="encodingFormatAC3">
           <property name="text">
            <string>AC3</string>
           </property>
          </widget>
         </item>
         <item row="0" column="2">
          <widget class="QCheckBox" name="encodingFormatEAC3">
           <property name="text">
            <string>EAC3</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QCheckBox" name="encodingFormatDTS">
           <property name="text">
            <string>DTS</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QCheckBox" name="encodingFormatMPEG">
           <property name="text">
            <string>MPEG</string>
           </property>
          </widget>
         </item>
         <item row="1" column="2">
          <widget class="QCheckBox" name="encodingFormatAAC">
           <property name="text">
            <string>AAC</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="offsetSelect" native="true">
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>&lt;b&gt;Latency offset:&lt;/b&gt;</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="offsetButton">
           <property name="suffix">
            <string> ms</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>advancedOptions</sender>
   <signal>toggled(bool)</signal>
   <receiver>advancedWidget</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>151</x>
     <y>103</y>
    </hint>
    <hint type="destinationlabel">
     <x>364</x>
     <y>122</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
